docs/install/npm.md

```diff
@@ -1,6 +0,0 @@
-# Install from NPM [not recommended]
-
-<!-- prettier-ignore-start -->
-!!! danger
```

Comment:
Might wanna keep this as a  warning so people don't install lodestar via npm

Reply:
Updated.

Im curious if we should not publish the package anymore to prevent against this.

Reply:
Could be installed as a package in another project in which case it would have a lock file again. I don't know if anybody is doing that though.

Reply:
There are 865 install / week listed on NPM.  Curious if that is all us and CI?

Reply:
>  Curious if that is all us and CI?

I have never installed this package, I don't see using doing that in the CI either, the README is definitely also out of date

Reply:
I read in a comment in our code in [`depgraph.md`](https://github.com/ChainSafe/lodestar/blob/959a8af1b757c46a0e57450788fb72773a372608/docs/design/depgraph.md?plain=1#L106).

I did a search on github and it seems the only references to `@chainsafe/lodestar` as a dependency are only very old forks.

https://github.com/search?q=%5C%22%40chainsafe%2Flodestar%5C%22%3A+path%3A**%2Fpackage.json+&type=code&ref=advsearch

Since its a security risk, perhaps we should stop publishing it and make it hidden for now and see who squeals that its gone.  What do you think @dapplion and @wemeetagain?

Reply:
I'd like to solve via https://github.com/ChainSafe/lodestar/issues/3596